src/sections/InvolvementSection.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { Box, Center, Wrap } from '@chakra-ui/react';
+import { graphql } from 'gatsby';
+import GetInvolvedCard from 'src/components/GetInvolvedCard';
+import SectionHeader from 'src/components/SectionHeader';
+
+interface Props {
+  data: GatsbyTypes.InvolvmentSectionFragment;
+}
+
+const InvolvmentSection = ({ data }: Props) => {
+  const involvementCards = data?.allContentfulGetInvolvedCard.nodes;
+
+  return (
+    <Box bg="gray.extralight" pb={20}>
+      <Box marginBottom={100}>
+        <SectionHeader text="Ways to get Involved" textPosition="right" />
+      </Box>
+      <Center>
```

Comment:
I think `Flex` might be preferred to `Center` and `Wrap` here - `Wrap` is usually used when we want to define a fixed space between children, but we don't need that in this case. `<Flex justify="space-evenly">` might be a good place to start

Reply:
I originally had it as `Flex` but wasn't sure if there was always going to be 3 items. Although I have breakpoints to change the direction to `column`, if we have more items in row, it will overflow horizontally.

Reply:
Yeah I think we can safely assume it's only three items